src/OperatorEnumConstruction.jl

```diff
@@ -248,6 +254,7 @@ function _extend_operators(operators, skip_user_operators, kws, __module__::Modu
         local $build_converters
         local $binary_exists
         local $unary_exists
+        lock($LATEST_LOCK)
```

Comment:
Could have a wrapper on SpinLock that simply warns if the lock is already locked